packages/variable-editor/src/components/variables/data/validation-utils.test.ts

```diff
@@ -1,49 +1,55 @@
 import type { ValidationMessages } from '../../../protocol/types';
+import { MockRow } from './test-utils/types';
 import { containsError, containsWarning, toValidationMessageVariant, validationMessagesOfRow } from './validation-utils';
+import type { Variable } from './variable';
 
 let validationMessages: ValidationMessages;
+let rowWithMessages: MockRow<Variable>;
+let rowWithoutMessages: MockRow<Variable>;
```

Comment:
why create this variables here, but add the values below?

Reply:
I saw this being done this way in other tests. I assumed it may have something to do with how the tests work or to guarantee that this data can never change between tests.

Can I instantiate these right here?

Reply:
I would say so, expect they change...